reporter/sources/php/exceptions.py

```diff
@@ -0,0 +1,101 @@
+import json
+
+from reporter.helpers import is_production_host
+from reporter.reports import Report
+
+from common import PHPLogsSource
+
+
+class PHPExceptionsSource(PHPLogsSource):
+    """
+    Report errors and exceptions logged via WikiaLogger
+    """
+    REPORT_LABEL = 'PHPExceptions'
+
+    FULL_MESSAGE_TEMPLATE = """
+h1. {exception}
+
+{message}
+
+h5. Backtrace
+{backtrace}
+"""
+
+    LIMIT = 50000
```

Comment:
Would be good to describe what is the limit for, either by adding a comment or making the name more descriptive